allocator/header.go

```diff
@@ -0,0 +1,19 @@
+package allocator
+
+import "unsafe"
+
+var (
+	headerSize = getHeaderSize()
```

Comment:
`const headerSize = int64(unsafe.Sizeof(header{}))`

Reply:
Nice catch, I'll fix this. Thank you!